qemu/tests/cfg/numa_opts.cfg

```diff
@@ -32,12 +32,13 @@
           extra_params += " -numa node -numa node"
           numa_node0_cpus = "0 2 4"
           numa_node0_size = 128
-          ppc64,ppc64le:
-              numa_node0_size = 768
           numa_node1_cpus = "1 3 5"
           numa_node1_size = 128
-          ppc64,ppc64le:
+          ppc64, ppc64le:
```

Comment:
Well, I tested this and found the issue was **NOT** caused by the different implementation of qemu between platforms. The thing happened here is two different results were gotten by different qemus.